client/src/pages/PostNewLease.js

```diff
@@ -14,6 +14,7 @@ import RadioGroup from '@mui/material/RadioGroup';
 import FormControlLabel from '@mui/material/FormControlLabel';
```

Comment:
Based on our current implementation, any changes in PostNewLease page layout should be reflected to the EdisLease page.